src/Compute/Compute/help/Get-AzVM.md

```diff
@@ -13,15 +13,10 @@ Gets the properties of a virtual machine.
 
 ## SYNTAX
 
-### ListAllVirtualMachinesParamSet (Default)
-```
-Get-AzVM [-Status] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
-```
-
-### ListVirtualMachineInResourceGroupParamSet
+### DefaultParamSet (Default)
 ```
-Get-AzVM [-ResourceGroupName] <String> [-Status] [-DefaultProfile <IAzureContextContainer>]
- [<CommonParameters>]
+Get-AzVM [[-ResourceGroupName] <String>] [[-Name] <String>] [-Status]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
```

Comment:
@maddieclayton it would be nice if we could add an example in this file (and other affected cmdlets) that uses the wildcard, as well as updating the description for the affected parameters letting users know that wildcards are accepted :shipit: